scripts/skills/actives/rf_pummel_skill.nut

```diff
@@ -122,8 +127,11 @@ this.rf_pummel_skill <- ::inherit("scripts/skills/actives/line_breaker", {
 		if (aoo == null)
 			return null;
 
+		if (this.m.RequiredWeaponType == null)
+			return aoo;
+
 		local weapon = aoo.getItem();
-		if (!::MSU.isNull(weapon) && weapon.isItemType(::Const.Items.ItemType.TwoHanded) && weapon.isItemType(::Const.Items.ItemType.Weapon) && weapon.isWeaponType(::Const.Items.WeaponType.Hammer))
+		if (!::MSU.isNull(weapon) && weapon.isItemType(::Const.Items.ItemType.Weapon) && weapon.isWeaponType(::Const.Items.WeaponType.Hammer))
```

Comment:
You could use the member variable `m.RequiredWeaponType` here as it also contains `Hammer`